source/content/guides/drupal/drupal-hosted-createcustom/11-troubleshooting.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Migrate a Custom Upstream to Drupal 9
+title: Migrate a Custom Upstream to the Latest Version of Drupal
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: <html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/jenniferhoffman/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/jenniferhoffman/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>
<!--table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
@page
	{margin:.75in .7in .75in .7in;
	mso-header-margin:.3in;
	mso-footer-margin:.3in;}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
br
	{mso-data-placement:same-cell;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:14.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	mso-number-format:General;
	text-align:general;
	vertical-align:bottom;
	border:none;
	mso-background-source:auto;
	mso-pattern:auto;
	mso-protection:locked visible;
	white-space:nowrap;
	mso-rotate:0;}
.xl65
	{color:#0563C1;
	text-decoration:underline;
	text-underline-style:single;}
-->
</style>
</head>

<body link="#0563C1" vlink="#954F72">



https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-createcustom/troubleshooting
--




</body>

</html>

Reply:
Must have missed them or merged them back in. Plz remove.